templates/slurm_stats.sh.epp

```diff
@@ -7,14 +7,17 @@
 
 source <%= $source_path %>
 
+## Pre-run cleanup
+rm -f /tmp/slurm_stats.*
+
 partitions=($("${slurm_path}"/sinfo -h | awk '{print $1}' | sort -u | xargs | sed 's/*//g'))
 
 ## Get data
-tf=$(mktemp /tmp/slurm1.XXXXXX)
-qf=$(mktemp /tmp/slurm2.XXXXXX)
-tfile3=$(mktemp /tmp/slurm3.XXXXXX)
-"${slurm_path}"/squeue -a -h -l > $tf
-"${slurm_path}"/squeue -h -l -O Partition,Username,State > $qf
+tfile1=$(mktemp /tmp/slurm_stats.XXXXXX)
+tfile2=$(mktemp /tmp/slurm_stats.XXXXXX)
+tfile3=$(mktemp /tmp/slurm_stats.XXXXXX)
```

Comment:
This worried me for a bit, but I confirmed that `mktemp` should never accidentally use a file that already exists.